test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -655,6 +655,8 @@ public final void testIndexTimeStoredFieldsAccess() throws IOException {
                 minimalMapping(b);
                 b.field("store", true);
             }));
+            assumeTrue("Mapper service is too old",
```

Comment:
Do we still need this check in the base class?

Reply:
Yes, we call the implementor for a service mapper and it can be returned theoretically in any version so we need to check, On the other hand we can force the implementor to return a specific version?

Reply:
It goes through `createFieldMapper(XContentBuilder)` which is overridden in LegacyGeoShapeFieldMapperTests though, so we should already be picking a working version?

Reply:
Yes but we are creating a working version with version < 8.0.0. For this test to work the version needs to be >= 8.0.0?

Reply:
https://github.com/elastic/elasticsearch/pull/72474 is merged so we should be able to remove this

Reply:
done